network-api/networkapi/wagtailcustomization/__init__.py

```diff
@@ -1,10 +0,0 @@
-from django.templatetags.static import static
```

Comment:
this one needs to stay where it is, the wagtailcustomization app is a special bit of code that overrides/presupplants some fo the functions and features that normal wagtail has.